.travis.yml

```
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "3.7"
```

I think this needs xenial

```suggestion
  - "3.7"

dist: xenial
```